src/web/app/pages-session/session-result-page/session-result-page.component.ts

```diff
@@ -183,6 +185,9 @@ export class SessionResultPageComponent implements OnInit {
             this.questions = sessionResults.questions.sort(
                 (a: QuestionOutput, b: QuestionOutput) =>
                     a.feedbackQuestion.questionNumber - b.feedbackQuestion.questionNumber);
+            if (this.questions.length === 0) {
+              this.isFeedbackSessionResultsEmpty = true;
```

Comment:
why create this new variable? We can just check for `this.questions.length == 0` in the html right?

Reply:
Yup, right. I didn't know about it before. I will change it.

Reply:
OK, I just realised in #11271, the issue occurred because the `*ngIf` is a bit buggy when straight up using `this.someArray.length === 0`. I just tested it out, and the same problem occurs here. I will change back to using this variable first, if you have any other way to solve this can let me know.

Reply:
You can change the condition to `!isFeedbackSessionResultsLoading && this.someArray.length === 0` to avoid the flushing issue.

Reply:
On a separate note, glad to see you back in action again @xpdavid Welcome back!

Reply:
Thanks @xpdavid ! The change above fixes this issue